ccdproc/image_collection.py

```diff
@@ -601,6 +603,7 @@ def _generator(self, return_type,
                    overwrite=False,
                    do_not_scale_image_data=True,
                    return_fname=False,
+                   ccd_kwargs={},
```

Comment:
Please change to `ccd_kwargs={}` to `ccd_kwargs=None` then, early in the function, add a line like 

```python
ccd_kwargs = ccd_kwargs or {}
```
or
```
if ccd_kwargs is None:
    ccd_kwargs = {}
``` 

Though the current implementation does not have any issues (because you are not modifying the dictionary between calls), in principle